tests/testUtil.ts

```diff
@@ -66,7 +67,27 @@ export const createAPIGatewayEvent = ({
     return eventTemplate
 }
 
-export const createAPIGatewayContext = (): Context => ({
+export const createSqsEvent = (...arns: string[]): SQSEvent => ({
```

Comment:
`createSQSEvent` seems more consistent with the rest of the project (uppercase abbreviations)